script/generate-cask.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env brew ruby
+require 'cask/cask'
+directories = ["_data/cask", "api/cask", "cask"]
+FileUtils.rm_rf directories
+FileUtils.mkdir_p directories
+
+Cask::Cask.each do |c|
+  json_filename = "#{File.basename(c.sourcefile_path,'.rb')}.json"
```

Comment:
Why `File.basename(c.sourcefile_path,'.rb')`? Isn't this the same as `c.token`?

Reply:
You are correct. I did not see this. I am updating it in my next commit.